spec/supabase_js_v2.yml

```diff
@@ -1564,7 +1564,7 @@ functions:
           ```ts
           const { data, error } = await supabase
             .from('users')
-            .upsert({ id: 42, handle: 'saoirse', display_name: 'Saoirse' })
+            .upsert({ id: 42, handle: 'saoirse', display_name: 'Saoirse' }, { onConflict: 'id' })
```

Comment:
based on the comment, I think it should be  `onConflict: 'handle'` right? I think we can also remove the `id` in this example, since it's illustrating an upsert without a PK

```
.upsert({ handle: 'saoirse', display_name: 'Saoirse' }, { onConflict: 'handle' })
```

Reply:
The description is a little confusing because it makes mention of the `id` column, so I wonder if we update the example if its going to confuse people even more.
> In the following query, `upsert()` implicitly uses the `id` (primary key) column to determine conflicts.

Reply:
From my understanding you have to leave the id for upsert to first check id against id and `{onConflict: 'handle'}` for it to check against that column after checking id first.

Reply:
@pucho I thought this too until I read v1 docs which doens't mention `id` at all and is more aligned with what @kiwicopple has suggested. https://supabase.com/docs/reference/javascript/v1/upsert